<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-9SAO9B6V" failures="2" tests="2" name="BankManagerLoginTest" time="1.32" errors="0" timestamp="2021-02-01T20:39:31 GMT">
  <testcase classname="com.w2a.testcases.BankManagerLoginTest" name="loginAsBankManager" time="0.477">
    <failure type="java.lang.AssertionError" message="login not successful">
      <![CDATA[java.lang.AssertionError: login not successful
at com.w2a.testcases.BankManagerLoginTest.loginAsBankManager(BankManagerLoginTest.java:19)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- loginAsBankManager -->
  <testcase classname="com.w2a.testcases.BankManagerLoginTest" name="loginAsBankManager" time="0.917">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;com.relevantcodes.extentreports.ExtentTest.log(com.relevantcodes.extentreports.LogStatus, String)&amp;quot; because &amp;quot;com.w2a.listeners.CustomListeners.test&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "com.relevantcodes.extentreports.ExtentTest.log(com.relevantcodes.extentreports.LogStatus, String)" because "com.w2a.listeners.CustomListeners.test" is null
at com.w2a.listeners.CustomListeners.onTestFailure(CustomListeners.java:28)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- loginAsBankManager -->
</testsuite> <!-- BankManagerLoginTest -->
